exp-player/addon/components/exp-card-sort/template.hbs

```diff
@@ -52,7 +59,20 @@
         <div class="row">
           {{#each group.categories as |category|}}
             <div class="col-sm-4 small-bucket">
-              <h5 class="text-center {{if (eq category.name 'qsort.sections.2.categories.neutral') '' 'bucket-label'}}">{{t category.name}}</h5>
+              <h5 class="bucket-label hidden-xs {{if (eq category.name 'qsort.sections.2.categories.neutral') '' 'bucket-label-margin'}}">{{t category.name}}</h5>
```

Comment:
See general note above: we should avoid using headings for cosmetic purposes. (also line 75 below)